data/wanderers.txt

```diff
@@ -425,6 +427,61 @@ phrase "friendly wanderer"
 	word
 		"."
 			
+phrase "friendly wanderer"
+	word
+		"Wanderer "
+		"Our "
+		"These "
+	word
+		"worlds"
+		"planets"
+		"places"
+		"lands"
+		"ecosystems"
+		"biospheres"
+	word
+		" are open to "
+		" are available to "
+		" welcome "
+	word
+		"anyone"
+		"anybody"
+		"any being"
+		"any lifeform"
+		"any [identity? consciousness? entity?]"
+	word
+		" "
+		" if they are "
```

Comment:
I think these sentences would be a bit hard to read without a comma here
ya could rephrase this word to have all options start with a comma, so ya don't have to read, say, 30 words without any pause